### DISCUSSION TITLE
Any way to reclaim this empty space in the task bar?

### DISCUSSION THREAD

**Discussion by s-h-a-d-o-w** [ramensoftware/windhawk-mods#1130] [Q&A] [answered]
(Also would be great to get rid of the "...")

![empty space](https://github.com/user-attachments/assets/136b40f7-b201-4d96-bf3c-34f2fdab594c)

**Comment by m417z** [comment, marked as answer]
There's no mod that improves this aspect of Windows. The [Taskbar Labels for Windows 11](https://windhawk.net/mods/taskbar-labels) mod has an option to reduce the minimum size of the buttons that's required for the overflow button to appear. I personally prefer seeing it as little as possible. Aside from the empty space issues, I don't find it to be very convenient.

**Reply by s-h-a-d-o-w** [reply]
Thanks!

(I don't understand what Windows devs were thinking here. What's the point of leaving that space empty? Especially when they render that "...". Surely they can calculate how much space is available and scale everything else up as much as possible. As it had been for decades.)

**Reply by s-h-a-d-o-w** [reply]
And just a clarify - while there is no mod currently, I guess you're also saying that there's no way to create one?

**Reply by m417z** [reply]
I'm not saying that there's no way to create one. There's no reason it wouldn't be possible, but it will require investing some time to understand how it's implemented and how a fix can be implemented.

**Reply by s-h-a-d-o-w** [reply]
Ah, cool!
Would you mind adding it to the "bounty board"? I was considering donating anyway, might as well put it towards that. 🙂

**Reply by m417z** [reply]
I assume you refer to the [Windhawk mods for the Windows 11 taskbar](https://ramensoftware.com/windhawk-mods-for-the-windows-11-taskbar) page. It's about 7+ Taskbar Tweaker features and a couple of major Windows 10 features that are no longer unavailable. I'm reluctant to add specific bugs there as well, that's what [Issues](https://github.com/ramensoftware/windhawk-mods/issues) are for.

How about we create an issue for it instead, then you can mention it in the [Recognition Wall](https://ramensoftware.com/donate#more-504) comments?

**Reply by s-h-a-d-o-w** [reply]
"I assume you refer to" - yeah, didn't realize that.

Otherwise - sounds good to me!

**Reply by m417z** [reply]
Great, I created an issue: https://github.com/ramensoftware/windhawk-mods/issues/1131.

**Reply by m417z** [reply]
Thank you for the donation, I appreciate it!

**Reply by s-h-a-d-o-w** [reply]
Don't mention it - I've been using your tools on and off for probably over a decade now. Awesome work!

(I think - when possible - we should all donate to small tools that we regularly use and throw less money at large corporations.)
